controlplane/configparser.cpp

```diff
@@ -197,6 +197,32 @@ controlplane::base_t config_parser_t::loadConfig(const std::string& rootFilePath
 	return baseNext;
 }
 
+void config_parser_t::loadConfig_route_bird(controlplane::base_t& baseNext,
+                                            std::vector<controlplane::route::bird_import_t>& birdsImport,
+                                            const nlohmann::json& birdJson)
+{
+	using BirdImport = controlplane::route::bird_import_t;
+	for (const auto& elemJson : birdJson)
+	{
+		BirdImport import;
+
+		if (exist(elemJson, BirdImport::socketStr))
+		{
+			import.socket = elemJson[BirdImport::socketStr];
+		}
+
+		if (exist(elemJson, BirdImport::vrfStr))
+		{
+			import.vrf = elemJson[BirdImport::vrfStr];
+		}
+
+		birdsImport.push_back(import);
+		YANET_LOG_INFO("loadConfig_route_bird: socket(%s), vrf(%s)\n",
+		               import.socket.data(),
+		               import.vrf.data());
+	}
```

Comment:
interchange those lines, and now we will be able to benefit from moving `import` object, which is essentially two `std::string`'s
```suggestion
		YANET_LOG_INFO("loadConfig_route_bird: socket(%s), vrf(%s)\n",
		               import.socket.data(),
		               import.vrf.data());
		birdsImport.push_back(std::move(import));
	}
```

Reply:
+